packages/govuk-frontend-review/src/stylesheets/full-page-examples/campaign-page.scss

```diff
@@ -1,11 +1,16 @@
 @import "govuk/base";
 @import "govuk/core";
 
-.app-header--campaign {
+.app-header--campaign,
+.govuk-feature--recolour .app-header--campaign {
   padding-bottom: govuk-spacing(2);
   border-bottom: $govuk-border-width-wide solid #fff500;
 }
 
+.govuk-feature--recolour .app-header--campaign {
```

Comment:
❤️ I like how having a global class enabling the feature allows us to easily update the app code to accomodate the flag (though I'd guess in a service, you'd either have or not have the flag enable so would need that less so).